src/web/mjs/connectors/ComicFuz.mjs

```diff
@@ -114,4 +126,20 @@ export default class ComicFuz extends Connector {
         const request = await this._createPROTORequest(uri, requestType, payload);
         return this.fetchPROTO(request, this.protoTypes, responseType);
     }
-}
\ No newline at end of file
+
+    convertWordArrayToUint8Array (wordArray) {
+        var len = wordArray.words.length,
```

Comment:
Is it possible to just change the `ArrayView` of the underlying byte buffer?
`return new Uint8Array(new Uint32Array(wordArray.words).buffer);`

Reply:
nvm. just tried it in browser and it seems the endianess does not match the words in CryptoJS (bytes for each word are reversed) :(